content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -2,85 +2,103 @@
 Start Receiving Tickets
 =======================
 
-Offering a variety of channels from where your customers can contact you grants them flexibility
-and the right to choose the best one for themselves. And, in order to make sure inquiries across
-all channels get addressed, it is essential to have a solution where all interactions come in one
-place.
+Helpdesk offers multiple channels where customers can reach out for assistance. In order to ensure
+all inquiries are addressed in a timely fashion, it is essential that all interactions are received
+in one central location.
 
-Channels options to submit tickets
+Channel options to submit tickets
 ==================================
 
-Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`, and enable the following
-features as you want them to be available to your users.
+Go to :menuselection:`Helpdesk --> Configuration --> Teams --> Channels` to select which
+channel options to enable. The available channels are *Email Alias*, *Website Form*, and
+*Live Chat*.
 
-.. image:: receiving_tickets/channels_options.png
+.. image:: receiving_tickets/receivingtickets_channeloptions.png
    :align: center
    :alt: View of a helpdesk teams setting page emphasizing the channels options in Odoo Helpdesk
 
 Email Alias
 -----------
 
-| Let your customers submit tickets by sending an email to your support email address. The subject
-  line of the email becomes the title of the ticket and the content is shown in the Chatter.
-| Select *Configure domain name* to be redirected to *Settings* and, from there, enable *External
-  Email Servers* to determine or change your *Alias Domain*.
+With this setting, users can create a ticket by sending an email to a custom email alias. The
+subject line of the email becomes the title of the ticket and the body of the message is added to
+the Chatter.
 
-.. image:: receiving_tickets/channels_emailalias.png
+Select :guilabel:`Configure a custom domain` to be redirected to :guilabel:`Settings` and, from
+there, enable :guilabel:`Custom Email Servers` to determine or change an :guilabel:`Alias Domain`.
+Click :guilabel:`Save`.
+
+.. image:: receiving_tickets/receivingtickets_emailalias.png
    :align: center
```

Comment:
Assuming we omitted the first screenshot (since was too generic), and we keep the content I added in previous comment, then I would use [this screenshot](https://i.imgur.com/6hFa2uj.png) instead.

- Includes other parts of UI for navigational context (not just an isolated block like an island)
- relevant to the text/content
- contains a highlight

For reference, I named that image `helpdesk-team-email-alias-example.png`, which is the correct naming format.